src/Form/Select.js

```diff
@@ -112,19 +116,25 @@ class Select extends Component {
   };
 
   render() {
-    const { id, name, options, placeholder, error, size = 'md', label, ...props } = this.props;
+    const { id, name, options, placeholder, error, size = 'md', label, floating, ...props } = this.props;
     const { value } = this.state;
+    const isFloating = floating && value != undefined && `${value}`.trim();
+    const StyledLabel = (
```

Comment:
nit: rename this to `FloatingLabel` so that is more obvious what this component is.